src/common/pure/ZeroApprovalWarning/ZeroApprovalWarning.tsx

```diff
@@ -8,7 +8,7 @@ const Link = styled(HashLink)`
 `
 
 interface ZeroApprovalWarningProps {
-  currency: Currency | undefined
+  currency: Currency | undefined | null
```

Comment:
Use the `Nullish` type https://github.com/cowprotocol/cowswap/blob/b4cdc4ed8a20408d985c6a55776b02905ae71661/src/types/index.ts#L11

Reply:
Makes sense - updating :+1: